### ISSUE TITLE
[FEATURE REQUEST] Output Emissions collection with lev positive up

### ISSUE THREAD

**Issue by lizziel** [geoschem/GCHP#115]
GCHP has always output the emissions diagnostic collection with levels inverted relative to all other diagnostic collections. Most emissions are 2D so this is not relevant for most emission outputs. However, for 3D emissions the convention difference is a pain for post-processing. It also can be a source of error if users are not aware of the difference from other diagnostics.

The lev positive down convention for HEMCO diagnostics originated in its use in GEOS which always stores and outputs variables with that convention. Changing it to lev positive up should be a straightforward change. The existing code for lev positive down should be kept but put into a MODEL_GEOS cpp ifdef.

**Comment by lizziel** [comment]
This feature request is somewhat related to:

- GCHP issue: https://github.com/geoschem/GCHP/issues/112
- MAPL feature request: https://github.com/GEOS-ESM/MAPL/issues/914

**Comment by lizziel** [comment]
If feature request https://github.com/GEOS-ESM/MAPL/issues/914 is implemented as proposed then no work would actually need to be done for this feature request beyond changing `HISTORY.rc` to specify `up` for Emissions collection. No changes to HEMCO would be required.

**Comment by lizziel** [comment]
Specifying lev positive up or down in history files is now possible in MAPL 2.8.3. When this will go into GCHP is still TBD.

**Comment by lfreese** [comment]
Hi @lizziel --Just checking in to make sure I've understood the relevant threads to this correctly (followed issues #112 and #10 here): if I'm using v 13.2.1, GEOSChem.Emissions collection output will have the surface at lev = 72, and toa at lev = 1; whereas all other collections (eg: SpeciesConc, AerosolMass, etc) have surface at lev = 1 and toa at lev = 72?

**Comment by lizziel** [comment]
Yes, that is correct! The restart file is also flipped relative to the other diagnostics.

**Comment by lizziel** [comment]
This update is being moved to the 14.1 release.

**Comment by lizziel** [comment]
I wonder if we should push this to 15.0 since it will break GCPy and maybe is more appropriate for a major release.

**Comment by lizziel** [comment]
We will include this feature in 15.0
